Rosyk_Task1.py

```diff
@@ -1,8 +1,5 @@
-print("Мультипарадигмальні мови програмування, Task 1")
+print("Multiparadigm programming languages, Task 1")
 print("Nikita Rosyk gr. IKM-221k, №15")
-for i in range(0, 45):
-    if i == 44:
-        print("Владислав Валерійович Овсяніков")
-    else:
-        print("Владислав Валерійович Овсяніков, ")
-print((11.2/(11.2-211.3))+(1.2/(15.2+11.3)))
\ No newline at end of file
+string = "Vladislav Valeriyovych Ovsyanikov"
```

Comment:
never call variables with built-in or standard words. a name of a variable should discribe its content, not a type.
list, string, str, dict, class -- is inappropriate.
let's consider the current case, you can use **full_name**, **teacher_name**